jujugui/static/gui/src/app/jujulib/index.js

```diff
@@ -57,9 +57,17 @@ var module = module;
       }
       callback(error, data);
     };
-    var failure = function(xhr) {
-      var data = JSON.parse(xhr.target.responseText);
-      var error = data.Message || data.message || data.Error || data.error;
+    var failure = function(xhrOrMessage) {
```

Comment:
Ah yes, good reminder to use `let` and `const` wherever possible. That said, since this is in jujulib (which I don't think is transpiled) I'm inclined to stick with `var` here. I'll add docblocks for both callbacks.